.github/workflows/update-license-years.yml

```diff
@@ -0,0 +1,20 @@
+name: Update copyright year(s) in license file(s)
+
+on:
+  schedule:
+    - cron: "0 3 1 1 *" # 03:00 AM on January 1
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  update-license-year:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: FantasticFiasco/action-update-license-year@v3
```

Comment:
Fwiw, I've been getting a bit paranoid about running too many github actions with write access, especially which don't come from the official github repos.

Reply:
That sounds fair. I'm not super familiar with GH actions so I don't fully grasp what the possibilities are here; is there a way around your concern using a less permissive configuration?

Reply:
> what the possibilities are here

possibilities are endless 😅 

For this bit, I don't think it's worth the risk of adding a new github action dependency for something we just need to do once a year. But I won't fight it :)